src/components/button-expand/ButtonExpand.tsx

```diff
@@ -0,0 +1,29 @@
+import clsx from "clsx";
+import React, { FC } from "react";
+import ExpandMoreIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/ExpandMore.svg";
+
+export type ButtonExpandProps = {
+  showMore: boolean;
+  setShowMore: (showMore: boolean) => void;
+};
+
+const ButtonExpand: FC<ButtonExpandProps> = ({ showMore, setShowMore }) => {
+  return (
+    <button
+      className="button-expand"
+      type="button"
+      onClick={() => setShowMore(!showMore)}
+      aria-label="Expand More"
```

Comment:
Please ensure that this text is translatable as it was the case in the original implementation.